lib/3scale/tasks/connectivity.rake

```diff
@@ -0,0 +1,29 @@
+namespace :connectivity do
+  desc 'Check connectivity of Redis Storage'
+  task :redis_storage_check do
+    begin
+      redis_instance = ThreeScale::Backend::Storage.instance
+      redis_instance.ping
+      puts "Connection to Redis Storage (#{redis_instance.id}) performed successfully"
```

Comment:
this should be out of the rescued code block, you can use an `else` block after `rescue` block (same for the other block below)